dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/StachUtilities.cs

```diff
@@ -34,6 +39,38 @@ public static string ValueToString(Value value)
             }
         }
 
+        /// <summary>
+        /// Returns the actual value object instead of the Protobuf Value object.
```

Comment:
Lets change this to "Returns the underlying raw object from the protobuf Value object"